packages/blade/src/components/Badge/Badge.tsx

```diff
@@ -134,7 +134,7 @@ const _Badge = ({
         paddingLeft={horizontalPadding[size]}
         paddingTop={verticalPadding[size]}
         paddingBottom={verticalPadding[size]}
-        display="flex"
+        display="inline-flex"
```

Comment:
`inline-flex` is not supported on react native. We'll have to add `isReactNative() ? 'flex' : 'inline-flex'` check here